Sitefinity CLI/CsProjModifier.cs

```diff
@@ -132,5 +137,21 @@ private static XElement GetFirstItemGroupXElementWithXElementsOfType(XDocument d
 
             return parent;
         }
+
+        private static string GetRelativePath(string destination, string origin)
+        {
+            Uri pathUri = new Uri(destination);
```

Comment:
could you place this declaration above its first usage i.e. ...result = Uri.UnescapeDataString